src/AttachmentPreview/AttachmentPreview.tsx

```diff
@@ -0,0 +1,111 @@
+import * as classnames from "classnames";
+import * as React from "react";
+import * as FontAwesome from "react-fontawesome";
+
+import { Button } from "../Button/Button";
+import { FlexBox } from "../flex";
+import { AttachmentFileType, FileAttachmentIcon } from "../MessagingAttachment/MessagingAttachment";
+
+import "./AttachmentPreview.less";
+
+export interface Props {
+  attachmentID: string;
+  attachmentName: string;
+  attachmentURL: string;
+  className?: string;
+  closeButtonAriaLabel?: string;
+  downloadButtonText?: string;
```

Comment:
As mentioned elsewhere, either this needs to be a required prop or we need an optional aria-label prop for when this one is absent!